.circleci/config.yml

```diff
@@ -162,8 +162,10 @@ jobs:
 workflows:
   check-and-deploy-development:
     jobs:
-      - check-code-formatting
-      - build-and-test
+      - check-code-formatting:
+          context: api-nuget-token-context
```

Comment:
It is needed in the code-format step otherwise the Nuget is not able to be authenticated. [See here](https://app.circleci.com/pipelines/github/LBHackney-IT/lbh-base-api/219/workflows/d6d07b87-1e76-4e61-8fca-9ae48f4dad63/jobs/413)